### PR TITLE
Handled null responses from search api

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-1004](https://team-1624093970686.atlassian.net/browse/AR-1004).

## Blocking dependencies

- [AR-778](https://team-1624093970686.atlassian.net/browse/AR-778)

## Changes

1. Renamed the function to searchUsersByWalletAddress
2. Added event listener for enter key press
3. In case of null return blank array instead of doing nothing. The null check was already added using optional chaining operator

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

**Review by mmjee — APPROVED** [review]
LGTM assuming tested 👍🏼

File: src/pages/AppUsers.vue [hunk 1]
```diff
@@ -420,6 +421,8 @@ export default {
                 actionCount: user.action_count,
               },
             ];
+          } else {
+            data.value = [];
```
Comment (ajithranka) [inline src/pages/AppUsers.vue:425]: Should we rename this ref from `data` to `users` for clarity?

**Review by ajithranka — CHANGES_REQUESTED** [review]
For later: We may need to include a clear button next to the search input to clear the search results. I'll write a note for the design team.

File: src/pages/AppUsers.vue (hunk 1 above)
Comment (shrinathprabhu) [reply src/pages/AppUsers.vue:425]: Yeah that makes sense

**Review by ajithranka — APPROVED** [review]
Looks good! 👌🏼

**Pull request merged**